Update production.py to comment out SECRET_KEY

carlrabeler42 opened the pull request:
## Purpose
<!-- Describe the intention of the changes being proposed. What problem does it solve or functionality does it add? -->
Commented out the SECRET_KEY setting as it is not needed for this web app and creates confusion with the Django web app that does require the SECRET_KEY setting.
* ...

## Does this introduce a breaking change?
<!-- Mark one with an "x". -->
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

<!-- Please check the one that applies to this PR using "x". -->
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
<!-- Add steps to run the tests suite and/or manually test -->
```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
<!-- Add any other helpful information that may be needed here. -->

carlrabeler42 committed:
Update production.py

pamelafox commented:
Flask apps also recommend a SECRET_KEY, documented here:
https://flask.palletsprojects.com/en/stable/config/#SECRET_KEY

carlrabeler42 committed:
Use environment variable for SECRET_KEY

added SECRET_KEY = os.getenv('SECRET_KEY') rather than embed value in the python file itself - avoid security bad practice and make the Flask container app and the Django container app consistent in this regard
